plugins/css-prefers-color-scheme/package.json

```diff
@@ -35,23 +47,21 @@
 		"CHANGELOG.md",
 		"LICENSE.md",
 		"README.md",
-		"browser.js",
-		"browser.min.js",
 		"dist"
 	],
 	"peerDependencies": {
-		"postcss": "^8.4"
+		"postcss": "^8.3"
```

Comment:
Any specific reason for this? Or is it just automatic change?

Reply:
Going to create a separate issue for this